sql/6_1_0-to-7_0_0_upgrade.sql

```diff
@@ -726,3 +726,13 @@ UPDATE `layout_options` SET `title` = 'Title' WHERE `layout_options`.`form_id` =
 #IfNotRow3D layout_options form_id DEM field_id fname title Name
 UPDATE `layout_options` SET `title` = 'Name', `titlecols` = '1', `datacols` = '3' WHERE `layout_options`.`form_id` = 'DEM' AND `layout_options`.`field_id` = 'fname';
 #EndIf
+
+INSERT INTO globals (gl_name,gl_index,gl_value) VALUES('enc_service_date',0,'show_both');
+INSERT INTO globals (gl_name,gl_index,gl_value) VALUES('enc_sensitivity_visibility',0,'show_both');
+INSERT INTO globals (gl_name,gl_index,gl_value) VALUES('enc_enable_issues',0,'show_both');
+INSERT INTO globals (gl_name,gl_index,gl_value) VALUES('enc_enable_referring_provider',0,'show_both');
+INSERT INTO globals (gl_name,gl_index,gl_value) VALUES('enc_enable_facility',0,'show_both');
+INSERT INTO globals (gl_name,gl_index,gl_value) VALUES('enc_enable_discharge_disposition',0,'show_both');
+INSERT INTO globals (gl_name,gl_index,gl_value) VALUES('enc_enable_visit_category',0,'show_both');
+INSERT INTO globals (gl_name,gl_index,gl_value) VALUES('enc_enable_class',0,'show_both');
+INSERT INTO globals (gl_name,gl_index,gl_value) VALUES('enc_enable_type',0,'show_both');
```

Comment:
is this needed?

Reply:
Whenever new vars are added to globals.inc until globals are saved then they are undefined. By adding the defaults then we are assured they are defined where used until user does a save in globals ui

Reply:
I usually do a coalesce to default in code whenever I add to globals for this very reason.

Reply:
thought upgrade or install script will cycle through globals.inc and add them to the db

Reply:
may be true because in the past I didn't run upgrade after including new globals so I made an assumption.

good to know so thanks Stephen!